### ISSUE TITLE
Training does not converge compared with Darknet Yolo

### ISSUE THREAD

**Issue by eyalzubia** [andy-yun/pytorch-0.4-yolov3#7]
Hi, I'm using the same configuration files (standard yolov3.cfg file) and data to train using this code, and with darknet. With darknet I get to a loss of <0.5 after a few hours, whereas in Pytorch the loss is at ~100. Moreover, in darknet I get good detection results and in Pytorch the detector does not seem to converge - it has many false positives and many misses. 
What am I missing? I thought the two implementations should be compatible.
Thanks very much!

**Comment by door5719** [comment]
@eyalzubia  ,I think your meaning is  training weights using darknet with c++ is faster than pytorch? I think python on windows is slow than c++(May be it is not right~).

**Comment by andy-yun** [comment]
Training process are almost  same on C++ and pytorch implementation. However, the initial value and other environments including losses  are not exactly same. Thus, you cannot get the same weight files from the same configuration file. But, if you adjust the learning rate and some configuration parameters, you will get the similar results. If you want to get more exact results, I recommend that you should train models on C++ original version.
